src/main/scala/viper/carbon/CarbonVerifier.scala

```diff
@@ -148,7 +161,13 @@ case class CarbonVerifier(private var _debugInfo: Seq[(String, Any)] = Nil) exte
             l.split(" ")
           case None =>
             Nil
-        })
+        }) ++
+          (config.counterexample.toOption match {
+            case Some(_) => {
+              List("/mv:-")
```

Comment:
I tried basically all the options; if I remember correctly, I ended up using "/mv" over "printModel" because, if you had more than one error (and I guess more than one thread in Boogie?), with "printModel", Boogie prints the different modes at the same time and they get mixed up (one line from one model, then two from another one, then one from the first, then three from a third model, ...) and are completely unusable. With "/mv", for some reason, it prints one after another and separates them properly.

Reply:
Ah ok, thanks for the explanation, then let's keep "/mv:-".